### ISSUE TITLE
Geocoding: copy all attributes to the results layer

### ISSUE THREAD

**Issue by anitagraser** [riccardoklinger/Hqgis#60]
As previously discussed on Twitter (https://twitter.com/underdarkGIS/status/1250823444603830272) it would be immensely helpful if the geocoding results would include the original layer attributes.
This new capability would be particularly worthwhile since it is hard to add a feature id to the original layer when the input data comes as a CSV because those cannot be edited, i.e. no columns can be added.

**Comment by GitHubMLee** [comment]
Retaining attributes would also be immensely helpful for isochrone analysis too!

**Comment by FranksGIS** [comment]
I also agree: It would be very helpful if geocoding results would include the original layer attributes !
